src/json.hpp

```diff
@@ -8538,9 +8161,8 @@ class basic_json
             result.m_type.bits.parsed = true;
 
             // 'found_radix_point' will be set to 0xFF upon finding a radix
```

Comment:
The comment needs to be adjusted to the fact that `found_radix_point` is a Boolean now ("set to 0xFF" makes no sense).